routes/seaice.py

```diff
@@ -92,3 +101,25 @@ def run_point_fetch_all_seaice(lat, lon):
         if hasattr(exc, "status") and exc.status == 404:
             return render_template("404/no_data.html"), 404
         return render_template("500/server_error.html"), 500
+
+
+@routes.route("/seaice/enddate/")
+def seaice_enddate():
+    """Get the latest date of sea ice concentration data available in the Rasdaman server.
+    Args:
+        None
+
+    Returns:
+        JSON-like dict of the latest year and month of sea ice concentration data
+
+    """
+    hsia_encodings = asyncio.run(get_seaice_metadata())
+    latest_date = hsia_encodings["ansi"][-1]
+
+    try:
+        valid_date = validate_seaice_timestring(latest_date)
+        return {"year": valid_date.year, "month": valid_date.month}
+    except ValueError as e:
+        print(e)
+        # Defaults to known good data if the latest date is invalid
+        return {"year": 2024, "month": 6}
```

Comment:
I'm not sure we want to do this: the app kinda lives or dies based on reliable API access.  Perhaps we make a ticket such that if THIS request fails in the client, the client says "Super sorry, I'm ruined" & hides the map/community selectors -- ?